backend/src/plugins/Tags/TagsPlugin.ts

```diff
@@ -217,6 +217,14 @@ export const TagsPlugin = zeppelinGuildPlugin<TagsPluginType>()("tags", {
 
         return "";
       },
+
+      isMention: input => {
+        if (typeof input !== "string") {
+          return "";
+        }
+
+        return input.match(/^<(?:@[!&]?|#)\d+>$/);
```

Comment:
If you're just expecting a boolean as the return type, [RegExp.test()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) is faster than [String.match()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match).

Reply:
Given JS's truthiness with strings I figured it was more versatile for it to return the mention instead of true. That still works as a boolean but also means it can act as a filter (e.g. only keep/return an arg if it is a mention).
Thinking more about this now it would be better to make it a boolean, not as a filter. At least given the naming. I'll change it to that now, thanks!